src/App.tsx

```diff
@@ -38,7 +45,9 @@ function App() {
       errorElement: <NotFound />,
       children: [
         { index: true, element: <Dashboard /> },
-
+        { path: 'UserList', element: <UserList/> },
```

Comment:
we need to be consistent when naming paths, why not name it `users-list , ads, bookings` as the previous paths.